app/models/language.rb

```diff
@@ -1,2 +1,3 @@
 class Language < ApplicationRecord
+  belongs_to :user
```

Comment:
You'll need to update these associations now that you've made them more complex.

For some guidance, read through this article from The Odin Project: https://www.theodinproject.com/courses/ruby-on-rails/lessons/active-record-associations